<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessesPage</class>
 <widget class="QGroupBox" name="ProcessesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../icon.png</normaloff>../../../icon.png</iconset>
  </property>
  <property name="title">
   <string>Типы процессов</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableWidget" name="t_processes">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Единицы измерений</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Комментарий</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="l_addProcess">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Добавить тип процесса</string>
       </property>
       <property name="buddy">
        <cstring>le_processName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_processName"/>
     </item>
     <item>
      <widget class="QPushButton" name="b_addProcess">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="l_fileLuaProcesses">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Файл сценария Lua</string>
       </property>
       <property name="buddy">
        <cstring>le_fileLuaProcesses</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_fileLuaProcesses">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_fileLuaProcesses">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_fileLuaProcessesEdit">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
